medusa/indexers/indexer_config.py

```diff
@@ -41,10 +41,6 @@
 EXTERNAL_IMDB = 10
 EXTERNAL_ANIDB = 11
 
-# mapping = {
```

Comment:
@OmgImAlexis @ratoaq2 I know I previously provided you with these dicts. But I kinda reused them here for the external id mappings. I've looked where you used it, and implemented alternative solutions. But maybe i've missed something. Better check this.